### DISCUSSION TITLE
CorpseFinder deleted all my apks

### DISCUSSION THREAD

**Discussion by wyv9** [d4rken-org/sdmaid-se#1513] [Q&A] [answered]
I've been using this app for like a year and now it just deleted 7.8gb of apks on my sd card? i never missed with the settings or anything and this is the first time this happens, is there's any reason why this happened?
![Screenshot_20241220-034557_SD Maid SE](https://github.com/user-attachments/assets/5d225799-aa9e-4910-9e59-36e018bdd80f)

**Comment by d4rken** [comment]
From a quick search this is the only entry I can find:

https://github.com/d4rken-org/sdmaid-se/blob/11ee4cdb924c74db5a4ebee95f93ba8b07e872a2/app/src/main/assets/clutter/db_clutter_markers.json#L11746-L11749

For SD Maid to suggest this, your "Include common names" option would have be activated. Is that the case?

![Screenshot from 2024-12-20 09-15-07](https://github.com/user-attachments/assets/b9614bbe-95bb-46f8-891a-01630949e5d0)

**Reply by wyv9** [reply]
i had the first option enabled
it also deleted the whole file not just apk files
it also deleted another file i had named "fonts"

**Reply by d4rken** [reply, marked as answer]
The only way I can reproduce it, is by enabling the second option.
Same for `fonts`. Requires the second option as it's marked as a too common name.
I'm out of ideas here. Please reproduce the issue (just scan results no need to delete)  and record a debug log of it, then I'll look into it more.

![Screenshot from 2024-12-27 18-30-46](https://github.com/user-attachments/assets/1f3bd2db-1af1-458e-b370-f6b8680a0863)

**Reply by d4rken** [reply]
Was this a manual action or via scheduler or via uninstall watcher?

**Reply by wyv9** [reply]
It was a manual action, anyways thanks i didn't know what these 2 options exactly do
